docker/test_env/geth2.go

```diff
@@ -0,0 +1,302 @@
+package test_env
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+
+	tc "github.com/testcontainers/testcontainers-go"
+	tcwait "github.com/testcontainers/testcontainers-go/wait"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	"github.com/smartcontractkit/chainlink-testing-framework/docker"
+	"github.com/smartcontractkit/chainlink-testing-framework/logging"
+)
+
+const (
+	GO_CLIENT_IMAGE_TAG      = "v1.13.4"
+	GETH_ETH2_EXECUTION_PORT = "8551"
+)
+
+type GethGenesis struct {
+	EnvComponent
+	ExecutionDir string
+	l            zerolog.Logger
+	t            *testing.T
+}
+
+type Geth2 struct {
+	EnvComponent
+	ExternalHttpUrl      string
+	InternalHttpUrl      string
+	ExternalWsUrl        string
+	InternalWsUrl        string
+	InternalExecutionURL string
+	ExternalExecutionURL string
+	ExecutionDir         string
+	consensusLayer       ConsensusLayer
+	l                    zerolog.Logger
+	t                    *testing.T
```

Comment:
@Tofel can we decouple testing.T from this docker wrapper? see https://github.com/smartcontractkit/chainlink-testing-framework/pull/764/files#r1389077799